test/ServerlessClamscan.test.ts

```diff
@@ -282,6 +282,44 @@ test('check bucket triggers and policies for source buckets ', () => {
   });
 });
 
+test('Check bucket triggers and policies for external bucket', () => {
```

Comment:
We should also have a test making sure that we are throwing an error when we have an imported bucket and we haven't accepted responsibility

Reply:
done!